infraestructure/datastore/db.go

```diff
@@ -0,0 +1,19 @@
+package datastore
+
+import (
+	"log"
+
+	// "github.com/jinzhu/gorm"
+	"gorm.io/driver/sqlite"
+	"gorm.io/gorm"
+)
+
+// NewDB returns the instance of the connection to sqlite
+func NewDB() *gorm.DB {
+	DBName := "config/goolang-bootcamp-2020.db"
```

Comment:
You could add this file path to the config file and, in the main, pass the value as a parameter to use it here.